Sources/GRPC/GRPCClientConnection.swift

```diff
@@ -171,7 +153,163 @@ open class GRPCClientConnection {
   }
 }
 
+// MARK:- Configuration structures
+
+/// A target to connect to.
+public enum ConnectionTarget {
```

Comment:
Move this inside `GRPCClientConnection`, or `GRPCClientConnection.Configuration`? Just an idea, might make more sense to keep it outside.

Reply:
We should be able to reuse this for the server so I think separate makes more sense.